pkg/ibm_plugin/README.md

```diff
@@ -24,4 +24,14 @@ To maximize flexibility in the deployment's scale we recommend increasing the qu
 - VPCs.
 - Global Transit Gateway connections by disabling (setting to 0) all non VPC connections.  
   A user may further increase the number of connections by reducing the number of Global Transit Gateways per region.
-  e.g., currently reducing the quota for TGW per region from 5 to 3 will raise max connections quota from 25 to 43.
\ No newline at end of file
+  e.g., currently reducing the quota for TGW per region from 5 to 3 will raise max connections quota from 25 to 43.
+
+### Testing
+Tests require setting environment variable `PARAGLIDER_IBM_RESOURCE_GROUP_ID`. 
```

Comment:
I think we should move  the resource group id from ~/.ibm/credentials.yaml to env var as well.. Let me open another issue for that.